### PR TITLE
Update latencyBasedRouting

### PR DESCRIPTION
**Pull request by Pradhyumna02**
Updating latency based routing to response after getting information from three servers
Test: To test latency based routing make the 'performLatencyRouting' flag to true

### CODE REVIEW

**Comment by rotosonic** [comment]
1+ looks good

File: public/lib/latencyBasedRouting.js [hunk 1]
```diff
@@ -90,28 +93,36 @@
         var self = this;
         //latencyHttpOnComplete
         var latencyHttpOnComplete = function (result) {
+            var latencySum = result.reduce(function (a,b) {
+                return a.time + b.time;
```
Comment (kemarc) [inline public/lib/latencyBasedRouting.js]: Is this a sum?

**Comment by kemarc** [comment]
+1

**Comment by rotosonic** [comment]
Can you update var cache buster at line 46   from Date.now() to Math.random()? Date.now() will cause 304 responses. Other than that 1 + ran locally to verify

File: public/lib/latencyBasedRouting.js [hunk 2]
```diff
@@ -47,14 +49,16 @@
      */
     latencyBasedRouting.prototype.getNearestServer = function () {
         var self = this;
-        var url = '/testServer?location=' + this.location;
+        var dataUrl = this.url + '?location=' + this.location;
```
Comment (rotosonic) [inline public/lib/latencyBasedRouting.js]: Add math.random queryString parameters. Noticed 304 responses earlier.

Comment (Pradhyumna02) [inline public/lib/latencyBasedRouting.js]: updated

**Pull request merged**